tslib/src/functions/check-pin-expiry/check-pin-expiry.service.ts

```diff
@@ -0,0 +1,14 @@
+import { ISqlService } from '../../sql/sql.service'
+
+export class CheckPinExpiryService {
+  private sqlService: ISqlService
+
+  constructor (theSqlService: ISqlService) {
+    this.sqlService = theSqlService
+  }
+
+  async process () {
+    const sql = `DELETE FROM [mtc_admin].[checkPin] WHERE pinExpiresAt < GETUTCDATE()`
```

Comment:
i did question why you didn't implement a dedicated data service, but this is leaner and the service is simple enough 👌